frontend/components/color_encoding_to_sound/PaletteColor.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SliderInstrument from "../instruments/SliderInstrument";
+
+class PaletteColor extends React.Component {
+    render() {
+        const selected = this.props.selected;
+        const color = this.props.color;
+        const styleString = `background-color: rgb(${color.r},${color.g},${color.b})`;
+        const id = this.props.id;
+        let style = {
+            backgroundColor: `rgb(${color.r},${color.g},${color.b}`,
+            height: "15px",
+            width: "100px"
+        };
+
+        if (selected) {
+            style = {
+                    ...style,
+                    boxShadow: "5px 5px 4px gray"};
+        }
+
+        return (<div>
+                <div
```

Comment:
An empty `<div>` with an `onClick` handler is a code smell: this should probably be a `<button>` instead, for accessibility's sake.